frontend/public/components/container.jsx

```diff
@@ -166,7 +166,7 @@ const Details = (props) => {
         <h1 className="co-section-title">Network</h1>
         <dl className="co-m-pane__details">
           <dt>Node</dt>
-          <dd><Link to={`/nodes/${pod.spec.nodeName}`}>{pod.spec.nodeName}</Link></dd>
+          <dd><Link to={`/k8s/cluster/nodes/${pod.spec.nodeName}`}>{pod.spec.nodeName}</Link></dd>
```

Comment:
Any reason not to use `ResourceLink`?

Reply:
It wasn't being used before, so not sure. I just fixed the URL.

Reply:
I guess my point is that using `ResourceLink` would avoid bugs like this and we should switch (unless there's a reason it doesn't work here). Or possibly use use `resourcePath` to create the path.

Reply:
We tended to use helper functions in the old console to keep urls consistent & avoid mistakes, not sure if that would be useful here? Its easy to miss this stuff.

Reply:
Updated to use `<ResourceLink>`.